src/EventStore.ClientAPI/Internal/EventStoreNodeConnection.cs

```diff
@@ -512,26 +512,23 @@ internal class EventStoreNodeConnection : IEventStoreConnection, IEventStoreTran
 
 			return subscription.Start();
 		}
-		/*
-
-		        public EventStorePersistentSubscription ConnectToPersistentSubscriptionForAll(
-		            string groupName,
-		            Action<EventStorePersistentSubscription, ResolvedEvent> eventAppeared,
-		            Action<EventStorePersistentSubscription, SubscriptionDropReason, Exception> subscriptionDropped = null,
-		            UserCredentials userCredentials = null,
-		            int? bufferSize = null,
-		            bool autoAck = true)
-		        {
-		            return ConnectToPersistentSubscription(groupName,
-		                SystemStreams.AllStream,
-		                eventAppeared,
-		                subscriptionDropped,
-		                userCredentials,
-		                bufferSize,
-		                autoAck);
-		        }
-		*/
 
+		// public EventStorePersistentSubscription ConnectToPersistentSubscriptionForAll(
+		// 	string groupName,
+		// 	Action<EventStorePersistentSubscription, ResolvedEvent> eventAppeared,
+		// 	Action<EventStorePersistentSubscription, SubscriptionDropReason, Exception> subscriptionDropped = null,
+		// 	UserCredentials userCredentials = null,
+		// 	int? bufferSize = null,
+		// 	bool autoAck = true)
+		// {
+		// 	return ConnectToPersistentSubscription(groupName,
+		// 		SystemStreams.AllStream,
+		// 		eventAppeared,
+		// 		subscriptionDropped,
+		// 		userCredentials,
+		// 		bufferSize,
+		// 		autoAck);
+		// }
```

Comment:
What that section of the code commented?

Reply:
The code was commented out previously, I've removed them now